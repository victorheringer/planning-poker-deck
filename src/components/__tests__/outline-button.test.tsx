import { render } from "@testing-library/react";
import OutlineButton from "../outline-button";

describe("OutlineButton", () => {
  it("should render correctly with all props true", () => {
    const tree = render(
      <OutlineButton block flat responsiveGrow />
    ).asFragment();
    expect(tree).toMatchSnapshot();
  });

  it("should render correctly with all props false", () => {
    const tree = render(
      <OutlineButton block={false} flat={false} responsiveGrow={false} />
    ).asFragment();
    expect(tree).toMatchSnapshot();
  });
});
