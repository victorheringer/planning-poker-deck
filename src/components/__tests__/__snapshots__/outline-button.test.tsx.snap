// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OutlineButton should render correctly with all props false 1`] = `
<DocumentFragment>
  <button
    class="sc-bdvvtL CIlWp"
  />
</DocumentFragment>
`;

exports[`OutlineButton should render correctly with all props true 1`] = `
<DocumentFragment>
  <button
    class="sc-bdvvtL hLnLZu"
  />
</DocumentFragment>
`;
